unity/CITools/BuildDriver/TestTargets.cs

```diff
@@ -6,8 +6,8 @@ namespace BuildDriver;
 [Flags]
 public enum TestTargets
 {
-    None = 1 << 0,
-    Embedding = 1 << 1,
-    CoreClr = 1 << 2,
+    None = 0,
+    Embedding = 1 << 0,
+    CoreClr = 1 << 1,
     All = ~0
```

Comment:
I needed to fix this on my branch otherwise the arg parsing wouldn't work correctly.